amnestywebsite/humanity-theme
# Enable HTML anchor in Details block

Ref: https://github.com/amnestywebsite/humanity-theme/issues/124#issuecomment-2194867709

**Steps to test**:
1. add a Details block
2. in settings, add a HTML anchor
3. visit the frontend
4. set the URI to include the anchor as the fragment
5. the page should jump down to your details block
